app/Resources/views/full/blog_post.html.twig

```diff
@@ -0,0 +1,41 @@
+{% extends 'pagelayout.html.twig' %}
+
```

Comment:
I just realized that another feature is missing: the canonical url
```
{% block canonical %}
    <link rel="canonical" href=ez_field_value(content, 'canonical_url')>
{% endblock %}
```
and add in the `<head>`:
```
{% block canonical %}
{% endblock %}
```